roles/confluent.variables/defaults/main.yml

```diff
@@ -255,7 +255,10 @@ zookeeper_jmxexporter_enabled: "{{jmxexporter_enabled}}"
 ### Port to expose prometheus metrics. Beware of port collisions if colocating components on same host
 zookeeper_jmxexporter_port: 8079
 
+## destination path for zookeeper jmx config file
 zookeeper_jmxexporter_config_path: /opt/prometheus/zookeeper.yml
+## source file for zookeeper jmx config file
+zookeeper_jmxexporter_config_file: "zookeeper.yml"
```

Comment:
```suggestion

### Path on Ansible Controller for Zookeeper jmx config file. Only necessary to set for custom config.
zookeeper_jmxexporter_config_source_path: "zookeeper.yml"
```

Can you use 3 #'s and change the variable as above?

We have a script that parses annotations with 3 #'s and writes docs into VARIABLES.md.
Please also run `python doc.py` and commit the VARIABLES.md file